subsys/bluetooth/audio/cap_internal.h

```diff
@@ -70,6 +71,18 @@ struct bt_cap_initiator_proc_param {
 	};
 };
 
+#if defined(CONFIG_BT_BAP_BROADCAST_ASSISTANT)
+struct cap_broadcast_reception_start {
+
+	bt_addr_le_t addr;
+	uint8_t adv_sid;
+	uint32_t broadcast_id;
+	uint16_t pa_interval;
+	uint8_t num_subgroups;
+	struct bt_bap_bass_subgroup subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
+};
+#endif /* CONFIG_BT_BAP_BROADCAST_ASSISTANT */
```

Comment:
```suggestion
#if defined(CONFIG_BT_BAP_BROADCAST_ASSISTANT)
struct cap_broadcast_reception_start {
	bt_addr_le_t addr;
	uint8_t adv_sid;
	uint32_t broadcast_id;
	uint16_t pa_interval;
	uint8_t num_subgroups;
	struct bt_bap_bass_subgroup subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
};
#endif /* CONFIG_BT_BAP_BROADCAST_ASSISTANT */
```

Didn't consider that we'd have to move this out of the union to pass it on as an argument, but of course that's required. 

It's OK as is, but you can also revert this if you like

Reply:
I think I like it this way, and it can be argued that the union is more readable if we move all structs outside of it